endpoints/getting-started-grpc/helloworld/README.md

```diff
@@ -0,0 +1 @@
+[This code's documentation lives on the grpc.io site.](http://www.grpc.io/docs/quickstart/python.html)
```

Comment:
removed the helloworld/ subdirectory and moved files to getting-started-grpc/ directory